consensus/cosipbft/mod.go

```diff
@@ -159,6 +186,27 @@ func (a pbftActor) Propose(p consensus.Proposal, players mino.Players) error {
 	return nil
 }
 
+func (a pbftActor) newPrepareRequest(prop consensus.Proposal,
```

Comment:
This is an example why it really depends on the situation because there I return a struct so the nil does not work. Sometimes it's nice to declare the object at the beginning and fill it step by step so you can return the said object if an error occurred.